explainer.md

```diff
@@ -1,624 +1,397 @@
-      [{ width: oldHeaderStyle.width,
-        height: oldHeaderStyle.height,
-        transform: oldHeaderStyle.transform }],
-      { duration: 1000,
-        pseudoElement: '::shared-container(header-id)' });
-  }
+document.documentTransition.prepare(async (transition) => {
+  // …Identify State-A elements to capture, then:
+  await updatePageStateSomehow();
+  // The page is now in State-B.
+  // …Identify State-B elements to capture, then:
+  transition.start();
 });
 ```
 
-# Alternatives Considered
-
-## Heirarchical Properties
-
-This proposal disallows a shared element to be nested inside another shared
-element. The restriction avoids the need to preserve the hierarchy of the shared
-elements and associated properties (transform, clip, effects inherited by
-descendents) when creating pseudo elements. This is a consideration for future
-iterations of the feature.
-
-## Container/Child Split
-
-One consideration is to render each shared element using a replaced element
-directly instead of creating a container element. The motivation behind this
-split is to provide a stacking context to cross-fade the content of old and new
-shared elements. This is necessary to ensure blending identical pixels is a
-no-op using
-[plus-lighter](https://drafts.fxtf.org/compositing/#porterduffcompositingoperators_plus_lighter)
-blending. While same-origin transitions could work around this, it enables
-future extensibility for cross-origin transitions where cross-fading identical
-images would be common.
-
-## Natively Supporting Animating Box Decoration CSS Properties
-
-An alternate approach to the setup described in
-[Animating Box Decoration CSS Properties](#live-animatable-properties) is to
-support this natively in the browser by introducing a new `content-element()`
-function. This function would behave similarly to the `element()` function
-except skipping the following properties when painting the element: box
-decorations and visual effects which generate a stacking context. The image
-would also be sized to the element's content-box (as opposed to the border-box
-used by the element() function). The motivation for supporting this natively
-would be to make these properties animatable instead of requiring developers to
-implement it themselves.
+# Cross-fading
+
+Cross-fading two DOM elements is currently impossible if both layers feature transparency. This is due to the default composition operation: black with 50% opacity layered over black with 50% opacity becomes black with 75% opacity.
+
+However, the [plus-lighter](https://drafts.fxtf.org/compositing/#porterduffcompositingoperators_plus_lighter) compositing operation does the right thing when isolated to a set of elements whose `opacity` values add to 1. The "transition part" wrapper performs this isolation.
+
+Allowing `mix-blend-mode` to be set to `plus-lighter` will enable developers to create real cross-fades between elements for this feature and elsewhere.
+
+# Relation to `element()`
```

Comment:
I made a suggestion above to include this within the "single image" mode and specify the modifications needed to element() to support that mode.